webapp/src/main/java/com/box/l10n/mojito/security/RequestLoggingConfig.java

```diff
@@ -0,0 +1,38 @@
+package com.box.l10n.mojito.security;
+
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.web.filter.CommonsRequestLoggingFilter;
+
+@Configuration
+public class RequestLoggingConfig {
+
+  @Value("${l10n.logging.requests.includesQueryString:false}")
+  public boolean includesQueryString;
+
+  @Value("${l10n.logging.requests.includesHeader:false}")
+  public boolean includesHeader;
+
+  @Value("${l10n.logging.requests.includesPayload:true}")
+  public boolean includesPayload;
+
+  @Value("${l10n.logging.requests.maxPayloadLength:10000}")
+  public int maxPayloadLength;
+
+  @Value("${l10n.logging.requests.maxPayloadLength:Request Data: }")
```

Comment:
This is wrong: `l10n.logging.requests.maxPayloadLength`

Reply:
Good spot